### PR TITLE
Print queue, initial implementation:

### PR DESCRIPTION
**Pull request by ikusteu**
* Create a (placeholder) PrintJob document mapping
* Create a PrintQueue class and set up the listening logic

### To test this out (manually): 
- Start up docker compose of the main app (librocco/web-client) 
- start the dev server (librocco/web-client)
- navigate to the app (localhost:5173) and, in the dev tools console, create a function `enqueue`:
```ts
// Initialise the counter
i = 0
function counter() { return i++ }

// Enqueue
function enqueue() {
  // window._db is defined (we've set it up so for e2e tests)
  return _db
    ._pouch
    .put({ _id: `v1/print_queue/printer-1/job-${counter()}`, printer_id: "printer-1", content: "Print This!" })
}
```
- start this program (`python .`) since the logic is in `__main__.py`
- run `enqueue()` from the browser a couple of times
- each time `enqueue()` is ran, the output of this program should display the document change

### CODE REVIEW

File: print_queue.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import time
+
+from constants import VERSION
+
+from print_job import PrintJob
+
+
+class PrintQueue(object):
```
Comment (silviot) [inline print_queue.py:8]: This used to be necessary, but it's not anymore in python 3.
```
class PrintQueue(object):
```
is equivalent to
```
class PrintQueue:
```

**Review by silviot — APPROVED** [review]
The code in general is very "pythonic" and looks good.

I'd add this method to the `PrintJob` class to improve readability:
```python
    def __str__(self):
        return f"<Print job for {self.printer_id}: {self.content}>"
```

It would also make sense to include a `setup.py` file to turn this into a Python package that can be easily installed and distributed. I can take care of this.

The code as is suffices to aid the client side of the receipt printing functionality.

**Comment by ikusteu** [comment]
@silviot Glad to heart that 😄 
Yeah, I thought about adding a `__str__` method, but didn't know what the good formatting convention would be and the default seemed good for now.

About `setup.py`: I thought that's what `__init__.py` is for and am still a bit confused as to how the build/install process works, so yeah, please do that and I'll definitely ask you to elaborate once it's done.

Finally, should we fork the repo into librocco at this point and continue there?

File: print_queue.py (hunk 1 above)
Comment (ikusteu) [reply print_queue.py:8]: Damn, I've merged already. I'll remove the unnecessary part in future work.

**Pull request merged**